src/custom/utils/operator/index.ts

```diff
@@ -151,17 +151,19 @@ function toApiAddress(address: string, chainId: ChainId): string {
 
 async function _getJson(chainId: ChainId, url: string): Promise<any> {
   let response: Response | undefined
+  let json
   try {
     response = await _fetchGet(chainId, url)
+    json = await response.json()
```

Comment:
I think this throws is the response doesn't have a body, doesn't it?

Reply:
shouldn't that be expected? i.e if no body = no data = error no?

Reply:
Yeah, but aren't you counting on the body to extract the error message?

For example, 404 would have no body, and still not (be expected to) throw.